src/fellows.ts

```diff
@@ -61,12 +102,9 @@ export const fetchAllFellows = async (
         continue;
       }
 
-      // @ts-ignore
-      const additional = fellowData.info.additional as
-        | [{ Raw: string }, { Raw: string }][]
-        | undefined;
+      const additional = fellowData.info.additional;
 
-      // If it does not have additional data (GitHub handle goes here) we ignore it
+      // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
       if (!additional || additional.length < 1) {
```

Comment:
I don't trust that this will always not be null, so I'll keep the check from before